website/client/src/components/tasks/spells.vue

```diff
@@ -86,17 +84,24 @@
     .drawer-container {
       left: auto !important;
       right: auto !important;
-      min-width: 60%;
     }
   }
 
   .drawer-slider {
-    margin-top: 1em;
+    margin-top: 15px;
+    margin-bottom: 15px;
+    width: 72px;
```

Comment:
In general Zeplin will always show absolute `width` and `height` but usually we try to avoid using them (except where needed like icons) because they prevent elements from expanding and collapsing based on their content. For example the Equipment drawer with this pr is looking like this 

<img width="1058" alt="Schermata 2020-10-20 alle 16 35 32" src="https://user-images.githubusercontent.com/589911/96602433-4d68d180-12f3-11eb-9062-799fe968d3e6.png">